.github/workflows/publish.yml

```diff
@@ -0,0 +1,24 @@
+name: Gradle Plugin Publish
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    name: Publish plugin to gradle plugins portal
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 12
+        uses: actions/setup-java@v1
+        with:
+          java-version: 12
+      - name: Grant execute permission for gradlew
+        run: chmod +x gradlew
+      - run: cat ./gradle.properties
+      - name: Publish plugins with gradle
+        env:
+          GRADLE_PUBLISH_KEY: ${{ secrets.PUBLISH_KEY }}
```

Comment:
I hope this will work.

Reply:
google brought me here for the exact same reason, and it did help, a lot. thank you